server/s3_motr_writer.cc

```diff
@@ -329,7 +329,8 @@ void S3MotrWiter::create_object(std::function<void(void)> on_success,
   s3_motr_api->motr_obj_init(&obj_ctx->objs[0], &motr_uber_realm, &oid_list[0],
                              layout_ids[0]);
   obj_ctx->n_initialized_contexts = 1;
-
+  // Caller will save pv id and other attributes retuned by entity create API
```

Comment:
You can add some more comment
ie if the flag is not passed then motr will fallback to older model of fetching...